### PR TITLE
add wildcard to dist in package.json

### PR DESCRIPTION
**Pull request by davidchase**

### CODE REVIEW

**Comment by briancavalier** [comment]
:+1:

**Comment by briancavalier** [comment]
Perhaps we should drop node 4 and add node 6?

**Comment by davidchase** [comment]
yeah thats not a bad idea... and node.js 5 is next to go [Farewell to Node.js v5, Preparing for v7](https://nodejs.org/en/blog/community/v5-to-v7/)

**Comment by briancavalier** [comment]
Cool.  Let's drop Node 4 in this PR--6 will be LTS soon enough.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "dist/hold.js",
   "typings": "type-definitions/hold.d.ts",
   "files": [
-    "dist/hold.js",
+    "dist/*",
```
Comment (briancavalier) [inline package.json]: `"dist"` will work as well.

**Pull request closed without merging**